### PR TITLE
Added exit status for uperf failures

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
Context: https://github.com/cloud-bulldozer/benchmark-operator/issues/773
Realted Wrapper PR: https://github.com/cloud-bulldozer/benchmark-wrapper/pull/439
Adding exit status for uperf failure cases. This change needs to be integrated in all the workloads that use sanfu for execution as a future action item.

### Fixes
Made code changes to return a non-zero exit in case of failures.

### Testing
Tested it integrating with benchmark-wrapper.

### Screenshots
 - <a href="https://drive.google.com/file/d/1BfU_-9pjQqJnyWILQ2YTmG1abLrNciaN/view?usp=sharing"></a>
 - <a href="https://drive.google.com/file/d/1EfReFELZbmhhLQNtn2FSfyMCkq5B1rBM/view?usp=sharing"></a>

### CODE REVIEW

**Review by mukrishn — APPROVED** [review]
LGTM

**Comment by mukrishn** [comment]
@vishnuchalla can you fix the conflict and update the failure task to use the common role `benchmark_state` like in PR #781

**Comment by vishnuchalla** [comment]
> @vishnuchalla can you fix the conflict and update the failure task to use the common role `benchmark_state` like in PR #781

Done. Resolved the conflicts and used benchmark_state

File: roles/benchmark_state/tasks/failure.yml [hunk 1]
```diff
@@ -9,6 +13,7 @@
       uuid: "{{ uuid }}"
       complete: True
       state: Failed
-      message: "Reconcile failed on {{ ansible_failed_task.name }} {{ ansible_failed_task.action }} \n ```{{ ansible_failed_result | to_nice_json }}```"
+    # message: "Reconcile failed on {{ ansible_failed_task.name }} {{ ansible_failed_task.action }} \n ```{{ ansible_failed_result | to_nice_json }}```"
```
Comment (dry923) [inline roles/benchmark_state/tasks/failure.yml:16]: Let's remove the old commented out line

Comment (vishnuchalla) [reply roles/benchmark_state/tasks/failure.yml]: Done

**Review by dry923 — APPROVED** [review]
LGTM

**Pull request merged**